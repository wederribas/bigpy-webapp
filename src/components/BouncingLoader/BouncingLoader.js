// @flow
import React from 'react'
import './styles.css'

export default function BouncingLoader() {
  return (
    <div className="bouncing-loader">
      <div />
      <div />
      <div />
    </div>
  )
}
