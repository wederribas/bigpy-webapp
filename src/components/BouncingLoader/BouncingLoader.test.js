import React from 'react'
import { render } from 'react-testing-library'
import 'jest-dom/extend-expect'

import BouncingLoader from './BouncingLoader'

test('Renders the animated loading dots and check if there are 3 dots', () => {
  const { container } = render(<BouncingLoader />)
  const renderedDotsCount = container.querySelector('div').children.length

  expect(container.firstChild).toMatchSnapshot()
  expect(renderedDotsCount).toEqual(3)
})
