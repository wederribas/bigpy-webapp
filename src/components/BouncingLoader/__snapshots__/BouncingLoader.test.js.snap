// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the animated loading dots and check if there are 3 dots 1`] = `
<div
  class="bouncing-loader"
>
  <div />
  <div />
  <div />
</div>
`;
